includes/Traits/Experimental_Check.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Trait WordPress\Plugin_Check\Checker\Experimental_Check
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Traits;
+
+/**
+ * Trait for stable checks.
+ *
+ * @since n.e.x.t
+ */
+trait Experimental_Check {
+	/**
+	 * Returns the checks stability.
+	 *
+	 * @return string
```

Comment:
See https://github.com/10up/plugin-check/pull/219#discussion_r1248146160:
```suggestion
	 * Returns the check's stability.
	 *
	 * @since n.e.x.t
	 *
	 * @return string One of the check stability constant values.
```

Reply:
Thanks @felixarntz updated